src/features/filter.ts

```diff
@@ -1,12 +1,34 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Status } from '../types/Status';
 
-const initialState = {
+interface Filter {
+  query: string;
+  status: Status;
+}
+
+const initialState: Filter = {
   query: '',
   status: 'all',
```

Comment:
The initial state for `status` is set to the string 'all'. Ensure that this value matches one of the `Status` enum values to maintain type safety and consistency.